packages/serializer/addon/-private/transforms/boolean.js

```diff
@@ -9,7 +9,7 @@ import Transform from './transform';
   The `BooleanTransform` class is used to serialize and deserialize
   boolean attributes on Ember Data record objects. This transform is
   used when `boolean` is passed as the type parameter to the
-  [attr](./model_package_named_exports?anchor=attr) function.
+  [attr](/ember-data/release/modules/@ember-data%2Fmodel/?anchor=attr) function.
```

Comment:
are we sure this page exists when consumed in the docs app? it doesn't exist in the yuidocs.

Reply:
Yes. It works testing locally.